static/js/admin/scannerqueryresult.js

```diff
@@ -1,5 +1,9 @@
 document.addEventListener('DOMContentLoaded', () => {
-  if (!document.body.matches('.change-list.model-scannerqueryresult')) {
+  let result_list = document.querySelector(
```

Comment:
If you don't change `result_list`, you may want to use `const` instead.

Reply:
https://github.com/mozilla/addons-server/pull/15576/commits/40844610cccf8a5e1120328ae59bbb1f4d897ec7